Sources/Sampler/abstract_sampler.hpp

```diff
@@ -39,7 +39,7 @@ class AbstractSampler {
   // Computes the derivative of the machine on the current visible
   // Using the lookUp tables if possible
   virtual AbstractMachine::VectorType DerLogVisible() {
```

Comment:
Maybe this function should be just removed

Reply:
Does this have to happen in this PR though?